app/controllers/best_practices_controller.rb

```diff
@@ -43,8 +47,14 @@ def create
   def update
     update_resource @best_practice, best_practice_params
 
+    location = if @best_practice.obsolete && !@best_practice.errors.any? && hide_obsolete_best_practices != '0'
```

Comment:
```suggestion
    location = if @best_practice.obsolete && @best_practice.errors.empty? && hide_obsolete_best_practices != '0'
```
Y aparte de eso, ponelo ya en una variable porque queda épico así.